src/workerd/api/BUILD.bazel

```diff
@@ -159,6 +205,8 @@ wd_cc_capnp_library(
     kj_test(
         src = f,
         deps = [
+            ":encoding",
+            ":util-iocontext",
```

Comment:
These tests need to depend on util-iocontext even though none of them directly use it, that could be avoided based on what I commented above